README.md

```diff
@@ -6,24 +6,22 @@ VisitAfrica
 * Rita HATEGEKIMANA
 * Diane NGOGA NIYODUKORERA
 #### Description:
- 
+ VisitAfrica App, is an application designed to help the people who want to make tour in different countires of Africa. The tourist can be able to book the plane ticket, accomodations and places to visit before the visit time.
 #### Technologies used
 * Flutter
 * Dart
 #### Setup instructions
 * There is no need to enter your personal credentials to access this APP.
-* You can clone this repository to make your own changes by Open terminal by Ctrl+ Alt + T, clone, Type code .to open vs code editor or use android studio. depending on which editor you are using!
+* You can clone this repository to make your own changes by Open terminal by Ctrl+ Alt + T, clone, Type code .to open vs code editor or use android studio. depending on which editor you are using, they can both work!
 * run flutter pub get to install dependencies
 * run flutter run to start
 ####  Contact information
 * Email:sumurerwawase620@gmail.com
```

Comment:
- Please add github profile links for these contributors insteadd of their emails